src/test/java/uk/ac/ebi/atlas/hcalandingpage/HcaHumanExperimentDaoIT.java

```diff
@@ -33,27 +34,51 @@ void setUp() {
 
```

Comment:
Please also add a test case which has multiple `organism_parts` in the set to check if the functionality is working correctly or not.

Reply:
Added test case.